inspirehep/modules/theme/templates/inspirehep_theme/format/record/authors/Author_HTML_detailed.html

```diff
@@ -34,7 +34,7 @@
       <div class="row">
         <div class="col-md-12 summary" id="detailed-header-top">
           <div class="col-md-2">
-            <img class="profile-image img-circle" src="{{ record['positions'][0]['email'] | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com' width="100%">
+            <img class="profile-image img-circle" src="{{ record['positions'][0].get('email', 'gravatar@inspirehep.net') | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com' width="100%">
```

Comment:
Might be that some author does not have any items in `positions` by the way, so this line could continue failing.

Also with @jacenkow we were looking into making the default image dynamic based on the initials of the author, a la google https://raw.githubusercontent.com/hellsan631/material-avatar/master/sample.png so I guess the trick of registering our email in gravatar would not work in that case.